Cocktails9/Cocktails9/BottomRow/BottomRowView.swift

```diff
@@ -0,0 +1,96 @@
+import SwiftUI
+
+struct BottomRowView: View {
+    @State var selectedTab = 0
+    
+    enum TabbedItems: Int, CaseIterable{
+        case cocktails = 0
+        case favorites
+        case profile
+        
+        var title: String{
+            switch self {
+            case .cocktails:
+                return "Cocktails"
+            case .favorites:
+                return "Favorites"
+            case .profile:
+                return "Profile"
+            }
+        }
+        
+        var iconName: String{
+            switch self {
+            case .cocktails:
+                return "wineglass.fill"
+            case .favorites:
+                return "star.fill"
+            case .profile:
+                return "person.fill"
+            }
+        }
+    }
+    
+    
+    
+    var body: some View {
+        NavigationStack {
+            ZStack(alignment: .bottom){
+                TabView(selection: $selectedTab) {
+                    HomeView()
+                        .tag(0)
+                    
+                    FavoritesView()
+                        .tag(1)
+                    
+                    ProfileView()
+                        .tag(3)
+                }
+            }
+        }
+        ZStack{
+            HStack{
+                ForEach((TabbedItems.allCases), id: \.self){ item in
+                    Button{
+                        selectedTab = item.rawValue
+                    } label: {
+                        CustomTabItem(imageName: item.iconName, title: item.title, isActive: (selectedTab == item.rawValue))
+                    }
+                }
+            }
+            .padding(6)
+        }
+        .frame(height: 70)
+        .background(Color.clear)
+        .cornerRadius(35)
+        .padding(.horizontal, 26)
+    }
+}
+
+extension BottomRowView{
+    func CustomTabItem(imageName: String, title: String, isActive: Bool) -> some View{
+        HStack(spacing: 10){
+            Spacer()
+            Image(systemName: imageName)
+                .resizable()
+                .renderingMode(.template)
+                .foregroundColor(isActive ? .black : .gray)
+                .frame(width: 20, height: 20)
+            if isActive{
+                Text(title)
+                    .font(.system(size: 14))
+                    .foregroundColor(isActive ? .black : .gray)
+            }
+            Spacer()
+        }
```

Comment:
you can apply foreground to HStack. This way text and icon will have the same color, but written in one place. It's easier to maintain and you actually don't need to check `isActive` inline for Text color since it's shown only when `isActive` is true.

```suggestion
        HStack(spacing: 10){
            Spacer()
            Image(systemName: imageName)
                .resizable()
                .renderingMode(.template)
                .frame(width: 20, height: 20)
            if isActive{
                Text(title)
                    .font(.system(size: 14))
            }
            Spacer()
        }
        .foregroundColor(isActive ? .black : .gray)
```